app/styles/_layout.scss

```diff
@@ -22,10 +22,11 @@
 
   @include respond-to('big') {
     border-radius: $big-border-radius;
-    box-shadow: 0 1px 5px $input-row-border-color;
     margin: -15px auto 0 auto;
     min-height: 420px;
     padding: 50px 40px 40px 40px;
+    transition: all 250ms cubic-bezier(.07,.95,0,1);
```

Comment:
FWIW at 250ms, the transition occurs so quickly you don't really get to see the easing.

Reply:
Instead of `all` this needs to be `box-shadow` ?

See https://css-tricks.com/almanac/properties/t/transition/